test/integration/unregistered_user_can_add_items_to_cart_test.rb

```diff
@@ -23,8 +23,34 @@ class UnregisteredUserCanAddItemsToCartTest < ActionDispatch::IntegrationTest
     assert page.has_content?("pour over")
     assert page.has_content?("it taste really good")
     assert page.has_content?("$4.00")
-    assert page.has_css?("img")
+    # assert page.has_css?("img")
     # And there should be a "total" price for the cart that should be the sum of all items in the cart
     assert page.has_content?("$8.00")
   end
+
+  test 'unregistered user can add the same item multiple time to the cart' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.